app/assets/javascripts/pages/user-credential-page.js.coffee

```diff
@@ -0,0 +1,26 @@
+class @UserCredentialPage
+  constructor: ->
+    editor = ace.edit("ace-credential-value")
+    editor.getSession().setTabSize(2)
```

Comment:
If chaining does not work, I think it would be worth storing `editor.getSession()` in a local variable since it is called at least 4 times.